libraries/integration/Microsoft.Bot.Builder.Integration.AspNet.Core/StreamingExtensions/StreamingExtensionRequestHandler.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Protocol;
+using Microsoft.Bot.Schema;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core.StreamingExtensions
+{
+    internal class StreamingExtensionRequestHandler : RequestHandler
+    {
+        public StreamingExtensionRequestHandler(BotFrameworkStreamingExtensionsAdapter adapter, IBot bot)
```

Comment:
Per our discussion, just make a new one of these BFSEA on each request

Reply:
👍